### ISSUE TITLE
debug mode

### ISSUE THREAD

**Issue by JonnyMqc** [Hexeption/MCP-Reborn#206]
editing the code while its running in debug mode doesnt work i have to recompile and re run it every time i make a change. does it not work or am i doing something wrong

**Comment by oliviathevampire** [comment]
Are you adding new methods/fields? If so, that is not supported.

**Comment by JonnyMqc** [comment]
no i am literally just changing a normal line, say like mc.player.chat(“yes”); to mc.player.chat(“no”); and it doesnt update in game

**Comment by Legion64** [comment]
Yes, I have the same situation. I just do a simple string edit but the compiled content doesn't change.

**Comment by gamerbross** [comment]
there is a solution?

**Comment by gamerbross** [comment]
download the single hotswap plugin
https://plugins.jetbrains.com/plugin/14832-single-hotswap
